perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
+                     'Number of objects to send and/or receive per stream. '
+                     'Only applies to the api_multistream scenario.',
+                     lower_bound=1)
+flags.DEFINE_integer('object_size_kb', 1,
```

Comment:
Yes, this PR does not include object size distribution support.

However, I realize that I don't want to accidentally get a flag into a released version of PKB that I will have to rename when distributions lands I think the best choice would be to give it a forward-compatible name. I like `object_storage_object_size`. Right now, it will accept a value that is a single size, and 100% of the objects will be that size. When distribution support comes, the user will be able to supply either a distribution or a single size, which will be interpreted as a point distribution.

I think having the convenience of not specifying a full distribution when you want 100% of objects to be the same makes sense anyway as a convenience feature, so I don't think this adds unnecessary technical debt. How does that sound?

As to your question about how that will play together with the "api_multistream" scenario, I am planning for each stream to choose objects from the distribution independently. I will document that when I add distribution support.